modules/network-observability-overview.adoc

```diff
@@ -0,0 +1,22 @@
+= Using Overview
+Overview displays the overall, aggregated metrics of the network traffic flow on the cluster. As an administrator, you can monitor the statistics with the available display options.
+
+== Working with Overview
+As an administrator, you can navigate to overview by using the following steps:
+
+. On the left navigation panel, click *Observe* → *Network Traffic*.
+. In the *Network Traffic* page, click the *Overview* tab.
+
+You can view the graphical representation of the flow rate statistics. You can configure the scope of each flow rate data by clicking the menu icon.
```

Comment:
We need to add an explanation of the filtering capabilities and how they affect the data visualized. This also includes selecting a time slot and usage of the autorefresh button.

Reply:
If you mean the quick filters, I am covering those under a separate PR.